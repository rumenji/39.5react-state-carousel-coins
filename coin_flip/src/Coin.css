.Coin {
  display: flex;
  justify-content: center;
  /* flex-direction: column; */
}

.Coin-image {
  border-radius: 8px;
  height: 200px;
}