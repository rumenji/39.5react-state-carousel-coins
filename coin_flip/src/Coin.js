import React from "react";
import "./Coin.css";



 function Coin({caption, src}) {

  return (
    <div className="Coin">
      <img className="Coin-image" src={src} alt={caption} />
    </div>
  );
}

export default Coin;
