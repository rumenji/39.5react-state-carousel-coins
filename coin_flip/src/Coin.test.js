import React from "react";
import '@testing-library/jest-dom'
import { render } from "@testing-library/react"
import Coin from "./Coin";


it("renders a card", function(){
    render(<Coin />);
});

it("matches card snapshot", function(){
    const {asFragment} = render(<Coin />);
    expect(asFragment()).toMatchSnapshot();
});

