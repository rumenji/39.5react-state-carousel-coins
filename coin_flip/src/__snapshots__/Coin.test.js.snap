// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches card snapshot 1`] = `
<DocumentFragment>
  <div
    class="Coin"
  >
    <img
      class="Coin-image"
    />
  </div>
</DocumentFragment>
`;
